lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
The only issue we could have here is a clash with an existing rspec command, if the user add the `logstash/bin` to his $PATH. (naturally we have the same issue with the plugin command)